src/value.rs

```diff
@@ -234,8 +272,20 @@ impl TryFrom<ValueSum> for i64 {
     }
 }
 
+impl From<ValueSum> for i128 {
+    fn from(value: ValueSum) -> Self {
+        value.0
+    }
+}
+
+impl From<NoteValue> for ValueSum {
+    fn from(value: NoteValue) -> Self {
+        Self(value.into())
+    }
+}
+
 /// The blinding factor for a [`ValueCommitment`].
-#[derive(Clone, Debug)]
+#[derive(Clone, Copy, Debug)]
```

Comment:
str4d ([Link](https://github.com/zcash/orchard/pull/471#discussion_r2542817988)): `ValueCommitTrapdoor` is a blinding factor that we shouldn't allow to be arbitrarily copied around the stack.
```suggestion
#[derive(Clone, Debug)]
```